basics/lists-and-tuples.md

```diff
@@ -274,6 +335,41 @@ but some people like to do it this way.
 >>>
 ```
 
+
+You can have nested tuples.
+
+```python
+>>> n = 1, 2, 3
+>>> n
+(1, 2, 3)
+>>> n[0]
+1
+>>> l = 'a', 'b', 'c'
+>>> l
+('a', 'b', 'c')
+>>> l[0]
+'a'
+>>> t = n, l 
+>>> t
+((1, 2, 3), ('a', 'b', 'c')) #The tuples n and l are nested
+>>> t[0]
+(1, 2, 3)
+>>> t[1]
+('a', 'b', 'c')
+>>> t[1][2]
+'c'
+>>> v = ([1, 2, 3], [3, 2, 1,[7, 8, 9]])
+>>> v
+([1, 2, 3], [3, 2, 1, [7, 8, 9]])
+>>> v[1]
+[3, 2, 1, [7, 8, 9]]
+>>> v[1][3]
+[7, 8, 9]
+>>> v[1][3][0]
+7
+```
+
+
```

Comment:
The most common way to nest tuples and lists is to have a list of tuples. For example, if you have names and phone numbers, it could be like `[('Akuli', '123'), ('luisPinedo', '456')]`. Maybe show that?